controllers/warehouse-controllers.js

```diff
@@ -17,7 +17,7 @@ const deleteWarehouse = async (req, res) => {
             .where({ id: warehouseID })
             .del()
 
-        res.status(200).json({message: 'Warehouse deleted'})
+        res.status(204).send
```

Comment:
Here I am seeing 'rest.status(204).send' only. Did you mean to end it with just send?

Reply:
Yeah I read the ticket again and it said to only send 204 with no message